### DISCUSSION TITLE
Alternative mode for sending switch commands?

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#64] [Editor App Feature Requests]
First off, the app seems very intuitive and fine at the first glimpse. Good work!

A thought: How about making it possible entering some "execute" mode, where for instance pressing the buttons on the device image would do the commands that the buttons are programmed for. 

In this mode the app would send PC/CC changes or do bank jumps and so on.

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by simonaglover95** [comment]
Planning to add something along these lines! :)
